GherkinSyncTool/GherkinSyncTool/Configuration/Config.cs

```diff
@@ -18,8 +18,7 @@ public string BaseDirectory
                 _directory = value;
             }
         }
-        public string TagIdPattern { get; set; } = "tc:";
-        public string TagId { get; set; } = "   @tc:";
+        public string TagIdPrefix { get; set; } = "   @tc:";
```

Comment:
this value is set also in apsettings.config, so it can have any value. Therefore, we should not use special logic during parsing of TagIdPrefix (like trimming)

Reply:
Fixed